k8s/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: k8s-web-hello
+spec:
+  type: LoadBalancer
+  selector:
+    app: k8s-web-hello
+  ports:
+  - port: 3333
+    targetPort: 3000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: k8s-web-hello
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: k8s-web-hello
+  template:
+    metadata:
+      labels:
+        app: k8s-web-hello
+    spec:
```

Comment:
search disruption budgets and understand why we would use them and add them (and understand their gotchas)